Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -646,6 +648,70 @@ void ADefiWalletCoreActor::SendEthAmount(int32 walletIndex, FString fromaddress,
   }
 }
 
+void ADefiWalletCoreActor::SendEthAmountAsync(
+    int32 walletIndex, FString fromaddress, FString toaddress, FString amount,
```

Comment:
should the wallet index be unsigned? I assume negative index would cause an exception?

Reply:
unreal only mapping with signed integers, i'll add an exception when wallet index is minus